crypto/param_build.c

```diff
@@ -391,13 +376,5 @@ OSSL_PARAM *OSSL_PARAM_BLD_to_param(OSSL_PARAM_BLD *bld)
 
 void OSSL_PARAM_BLD_free_params(OSSL_PARAM *params)
```

Comment:
Let's remove this function:

- It's new, so there is no legacy requirements.
- It's a duplicate of another function.

Reply:
Yes - I was thinking the same thing..